iothub-manager/app/com/microsoft/azure/iotsolutions/iothubmanager/webservice/v1/controllers/DeploymentsController.java

```diff
@@ -112,6 +112,10 @@ public DeploymentsController(final IDeployments deploymentsService) {
             throw new InvalidInputException("Name must be provided");
         }
 
+        if (StringUtils.isEmpty(deployment.getConfigType())) {
+            throw new InvalidInputException("Config Type must be provided");
```

Comment:
Config Type must be provided -> Configuration Type must be provided

Reply:
done